cpp/source/stats/include/MetricBidiReactor.h

```diff
@@ -36,8 +36,8 @@ class MetricBidiReactor
     : public grpc::ClientBidiReactor<ExportMetricsServiceRequest,
                                      ExportMetricsServiceResponse> {
 public:
-  MetricBidiReactor(std::weak_ptr<Client> client,
-                    std::weak_ptr<OpencensusExporter> exporter);
+  MetricBidiReactor(std::shared_ptr<Client> client,
```

Comment:
Why shared? weak_ptr shall suffice.

Reply:
This is to fix the crash happened in MetricBidiReactor.